pyproject.toml

```
@@ -1,2 +1,11 @@
 [build-system]
-requires = ["pip>=18", "setuptools>=40", "wheel"]
+requires = ["pip>=19.3", "setuptools>=40", "wheel"]
```

maybe?
```suggestion
requires = ["pip>=19,!=19.1", "setuptools>=40", "wheel"]
```
It looks like pip 19.2 dropped support for Python 3.4.

---

I'm not sure if pip is a build system requirement at all.
`python setup.py install` works without `pip`; it requires `setuptools` only.

Maybe we can drop `[build-system]` entirely, the implicit fallback to `setuptools` "just works".

---

No. We shouldn't call the back-end directly.

---

Please elaborate, do you agree with dropping `pip` or the entire section?

---

Reverted the line; the change worth a separate PR

---

I don't think we should touch it now.